packages/terra-data-grid/src/WorklistDataGrid.module.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable terra/custom-property-namespace */
```

Comment:
Should the classnames in this file be updated rather than disabling this rule? Is that something that will be done in a future story?

Reply:
I was thinking these might need to be updated, but also to avoid breaking this rule, the names might get long/redundant (ex: `--terra-data-grid-worklist-data-grid-cell-focus-outline`). If we do prefer these be renamed, it should probably be done as part of this story.

Reply:
We can look at refining the variable names when we refactor the implementation. But to simply changes, we should stick to renaming the package and imports in this PR/story.